app/yarn.lock

```diff
@@ -3186,7 +3186,7 @@ event-target-shim@^5.0.0, event-target-shim@^5.0.1:
 
 eventemitter3@^4.0.7:
   version "4.0.7"
-  resolved "https://registry.npmjs.org/eventemitter3/-/eventemitter3-4.0.7.tgz"
+  resolved "https://registry.yarnpkg.com/eventemitter3/-/eventemitter3-4.0.7.tgz#2de9b68f6528d5644ef5c59526a1b4a07306169f"
```

Comment:
Wondering why this changed. Did you checkout the lockfile from `3rd party auth` or from `main`? Let me see what's in `main`...